### ISSUE TITLE
Windows app SMB share issue.

### ISSUE THREAD

**Issue by lthieman** [netbirdio/netbird#2199]
**Describe the problem**

Initially after installing the Netbird app on my Windows 11 PC everything works fine for hours or sometimes days, then my SMB shares from my TrueNAS server, which is on the local network, will become inaccessible from my Windows PC. Sometimes I get "invalid path" sometimes not. Disconnecting the Netbird app does not correct the issue, fully uninstalling the app does. 
 
**To Reproduce**

1. Install Netbird on Windows 11 PC
2. Wait 1hr-3days
3. Attempt to access local network TrueNAS SMB shares with File Explorer

**Expected behavior**

I expect Windows not to be confused about how to access SMB shares despite using Netbird.

**Are you using NetBird Cloud?**

Yes, Netbird Cloud. 

**NetBird version**

Several versions over the last couple months, 28.2 is the most recent I tried and it still exhibited this behavior 

**NetBird status -d output:**

This is above my skillset, sorry. 

**Additional context**

I'm eagerly awaiting being able to install Netbird on my PfSense router as I think that will just avoid the issue altogether. 

Thanks for you time!

**Comment by bcmmbaga** [comment]
Hello @lthieman, could you please share the client logs found in `C:\ProgramData\Netbird\client.log` while we attempt to reproduce the issue on our end?

**Comment by lthieman** [comment]
> Hello @lthieman, could you please share the client logs found in `C:\ProgramData\Netbird\client.log` while we attempt to reproduce the issue on our end?

I uninstalled the app to get my shares working again so I don't have that data anymore but I will reinstall it and share those logs once the shares stop working again.

**Comment by v1rusnl** [comment]
For me it's the other way around. When using the Netbird client on my Windows 11 23H2, after some time the enabled Windows network shares on the machine aren't accessible anymore from other clients in LAN, e.g. Android File Browsers or Debian Micro Server. They just are reachable for other Netbird peers under the Netbird IP. As soon as I stop Netbird client UI and deactivate the service followed by a reboot, the shares are accessible again in LAN.

If you need any logs or if I should open an own issue, please tell me. I use selfhosted Netbird, latest version.

**Comment by nazarewk** [comment]
Hello @lthieman,

We're currently reviewing our open issues and would like to verify if this problem still exists in the [latest NetBird version](https://github.com/netbirdio/netbird/releases).

Could you please confirm if the issue is still there?

We may close this issue temporarily if we don't hear back from you within **2 weeks**, but feel free to reopen it with updated information.

Thanks for your contribution to improving the project!

**Comment by lthieman** [comment]
> Hello [@lthieman](https://github.com/lthieman),
> 
> We're currently reviewing our open issues and would like to verify if this problem still exists in the [latest NetBird version](https://github.com/netbirdio/netbird/releases).
> 
> Could you please confirm if the issue is still there?
> 
> We may close this issue temporarily if we don't hear back from you within **2 weeks**, but feel free to reopen it with updated information.
> 
> Thanks for your contribution to improving the project!

Yes, on the latest version it seems to no longer be an issue. Accessing the share seems to be a little more laggy with Netbird running but it works and doesn't disappear.

**Comment by Nordlicht-13** [comment]
I still have this issue, but I can disconnect NetBird and connect again
and than I can access the smb-shares again.

**Comment by TechHutTV** [comment]
Per the reporter's confirmation on, the disconnecting-SMB-share symptom no longer reproduces on the latest NetBird version. Closing; please reopen with logs if shares become inaccessible again on a current release.

@Nordlicht-13 do you still have issues?

**Comment by Nordlicht-13** [comment]
@TechHutTV After my excursion to headscale/tailscale I'm back on NetBird again. The SMB-Share Problem is gone, thanks.

**Comment by Nordlicht-13** [comment]
@TechHutTV The SMB-Shere Problem is gone, but my apps on my truenas (e.g.  searxng) are just usable after disabling and enabling NetBird. I use ngnix for the app URLs.
